02 Algorithm Reference/06 Securities and Portfolio/01 Introduction.html

```diff
@@ -48,8 +48,8 @@
 </pre>
 	<pre class="python">// Popular Portfolio Property Values:
 self.Portfolio.Invested                // Hold at least one stock
-              .Cash                    // Sum of all currencies in account (only unsettled cash)
-              .UnsettledCash           // Sum of all currencies in account (only settled cash)
+              .Cash                    // Sum of all currencies in account (only settled cash)
```

Comment:
@StefanoRaggi is this actually "(including unsettled cash)"?

Reply:
`Portfolio.Cash` only contains settled cash and `Portfolio.UnsettledCash` only contains unsettled cash.

There are two separate cashbooks: `Portfolio.CashBook` and `Portfolio.UnsettledCashbook`.
At settlement time, funds are automatically moved from `Portfolio.UnsettledCashbook` to `Portfolio.CashBook`.

> Portfolio.Cash = Portfolio.CashBook.TotalValueInAccountCurrency
> Portfolio.UnsettledCash = Portfolio.UnsettledCashbook.TotalValueInAccountCurrency